tests/conversion_script/test_main.py

```diff
@@ -71,16 +80,17 @@ def test_main_success_c2r_installed(
     assert mock_update_insights_inventory.call_count == 1
     assert mock_gather_json_report.call_count == 1
     assert mock_find_highest_report_level.call_count == 1
-    assert mock_gather_textual_report.call_count == 1
-    assert mock_generate_report_message.call_count == 1
+    assert mock_gather_textual_report.call_count == 0
+    assert mock_generate_report_message.call_count == 0
     # NOTE: we should expect below one call once we don't require rpm because of insights conversion statistics
-    assert mock_cleanup_pkg_call.call_count == 0
+    assert mock_cleanup_pkg_call.call_count == 1
     # NOTE: successful conversion keeps gpg and repo on system (the backup is also kept)
-    assert mock_cleanup_file_exists_call.call_count == 0
-    assert mock_cleanup_file_restore_call.call_count == 0
+    assert mock_cleanup_file_exists_call.call_count == 4
+    assert mock_cleanup_file_restore_call.call_count == 2
     assert mock_transform_raw_data.call_count == 1
     assert mock_get_system_distro_version.call_count == 1
     assert mock_is_non_eligible_releases.call_count == 1
+    assert mock_archive_analysis_report.call_count == 0
```

Comment:
I don't know if this matches with expectations :thinking: the original values are expected, in case of successful conversion and c2r installed there shouldn't be any cleanup.. ah I am not sure now if the tests name doesn't match or what is the problem